src/instruments/src/EFB/Assets/320neoOutline [Upright].svg

```diff
@@ -0,0 +1,3295 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Please do not put spaces in filenames

Reply:
I followed the convention of existing files in this directory, do you want me to change all of them?

Reply:
I went ahead and updated the image names and all of the references of those images. In future, I think it would be a good idea to document style preferences like this, since all I had to go on was what was already in the code.